packages/react-server-test-pages/pages/data/delay.js

```diff
@@ -8,8 +11,8 @@ export default class DelayDataPage {
 		return 'application/json';
 	}
 	getResponseData() {
-		const { ms, val } = this.getRequest().getQuery();
+		const { ms, val, big } = this.getRequest().getQuery();
```

Comment:
Added a `big=<size>` param to the test data endpoint that just tells it to return a JSON object with `size` keys.  It's easiest to see the fold when there's a bunch of JS to for the browser to slurp up right after.